pkgs/servers/meteor/default.nix

```diff
@@ -62,7 +62,7 @@ stdenv.mkDerivation rec {
     for p in $(find $out/packages -name '*.node'); do
       patchelf \
         --set-rpath "$(patchelf --print-rpath $p):${stdenv.cc.cc.lib}/lib" \
-        $p
+        $p || true
```

Comment:
Several binaries in the new meteor tar are not patch-able. I add this so those failed binaries get ignored and the build passes.